Code/autopkglib/PkgPayloadUnpacker.py

```diff
@@ -81,14 +83,44 @@ def unpack_pkg_payload(self):
             )
             (_, err_out) = proc.communicate()
         except OSError as err:
-            raise ProcessorError(
+            error = (
                 f"ditto execution failed with error code {err.errno}: {err.strerror}"
             )
         if proc.returncode != 0:
-            raise ProcessorError(
+            error = (
                 f"extraction of {self.env['pkg_payload_path']} with ditto failed: "
                 f"{err_out}"
             )
+        if error and os.path.exists("/usr/bin/aa"):
+            try:
+                unpack_cmd = [
+                    "/usr/bin/aa",
+                    "extract",
+                    "-i",
+                    self.env["pkg_payload_path"],
+                    "-d",
+                    self.env["destination_path"],
+                ]
+                proc = subprocess.Popen(
+                    unpack_cmd,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.PIPE,
+                    text=True,
+                )
+                proc.communicate()
```

Comment:
Please prefer subprocess.run() instead of Popen.

Reply:
Do you have another example from within `autokglib` so I can follow the style? All I did was copy the existing code for the `ditto` `proc` command.

Reply:
@grahampugh Some other examples:
https://github.com/autopkg/autopkg/blob/7ba6e9da183683e9991882b99cba0d30d359bda8/Scripts/bulk_repo_add.py#L79

https://github.com/autopkg/autopkg/blob/71a1737859ead371aebb2ffcfdbf33e2a4557d80/Code/autopkglib/URLGetter.py#L169